### DISCUSSION TITLE
Using latest commit even if CI is KO?

### DISCUSSION THREAD

**Discussion by daudrain** [google/googletest#3778] [Community Help]
Hello,

The documentation says _We recommend using the latest commit in the main branch in your projects_ but I do not see any CI success.

Do you still recommend to use latest commit even if CI is KO?

Thank you,
David

**Comment by dinord** [comment]
We use Kokoro CI (Google's internal tool) as the official CI, and it's currently green. GitHub Actions are currently a best-effort CI maintained strictly by community contributions.

Your question does raise two valid issues:
1. The landing page does not show the status of Kokoro CI
2. There is no documentation that indicates GHA is best-effort
